src/ch02-08-foundry-cast.md

```diff
@@ -17,18 +17,24 @@ In this section, we'll delve into `sncast`.
 
 ## Requirements
 
-```txt
-scarb 2.3.0 (f306f9a91 2023-10-23)
-cairo: 2.3.0 (https://crates.io/crates/cairo-lang-compiler/2.3.0)
-sierra: 1.3.0
-snforge 0.10.1
-sncast 0.10.1
+```bash
+# scarb --version
+scarb 2.4.3
+cairo: 2.3.0
+sierra: 1.4.0
+
+# snforge --version
+snforge 0.14.0
+
+# sncast --version
+sncast 0.14.0
 The Rust Devnet
 ```
 
 ## Step 1: Sample Smart Contract
 
-The following code sample is sourced from `starknet foundry`. You can find the original [here](https://foundry-rs.github.io/starknet-foundry/testing/contracts.html).
+The following code sample is sourced from `starknet foundry`(You can find the original [here](https://foundry-rs.github.io/starknet-foundry/testing/contracts.html)).
+Also if yo desire to directly clone you can do it from this [Github REPO](https://github.com/machuwey/foundry_example.git)
```

Comment:
Please can you add your repo to our `examples` directory https://github.com/starknet-edu/starknetbook/tree/main/examples

Then you have update the link to use of the repo on examples

Reply:
Do you mean put it as a subrepo or just copy the raw files from my repo? Thanks in advance

Reply:
Just copy the raw files. No submodule/subrepo

Reply:
> Just copy the raw files. No submodule/subrepo

Okey, [d000ff9](https://github.com/starknet-edu/starknetbook/pull/307/commits/d000ff971695de362c5102d49f2e9db88e20692e) should do it